test/blackbox-tests/utils/dune

```diff
@@ -22,3 +22,17 @@
 (executable
  (name refmt)
  (modules refmt))
+
+(executable
+ (name dune_console_styles)
+ (modules dune_console_styles)
```

Comment:
what about juts using an expect test rather than introducing a dummy binary?

Reply:
I prefer the binary because then I can run that manually.

Reply:
Okay, that’s not enough of a reason. You can run inline tests manually as well.